xblockmufi/private/edit.js

```diff
@@ -34,4 +36,33 @@ function XblockMufiEdit(runtime, element) {
         });
         return false;
     });
+
+    // TEXT EDITOR JS
+    $('li.specialButton').click(function() {
+        $(this).toggleClass('active');
+        $('.sr .toggle-open', this).toggle();
+        $('.specialCharTable', $element).toggle();
+    });
+
+    $('.txtEditor li.styleButton').mousedown(function(){
+        var command = $(this).data('command');
+        var argument = $(this).data('argument');
+        document.execCommand(command, false, argument);
+        $('.contents', $element).focus();
+    });
+
+    $('.specialCharTable .charWrapper .char').hover(function() {
```

Comment:
Here and throughout, can we refer to these `specialCharacter`, assuming that's the intent of the abbreviation?